hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/ListingBasedRollbackStrategy.java

```diff
@@ -103,6 +103,10 @@ public List<HoodieRollbackRequest> getRollbackRequests(HoodieInstant instantToRo
       if (commitMetadataOptional.isPresent()) {
         isCompaction.set(commitMetadataOptional.get().getOperationType() == WriteOperationType.COMPACT);
       }
+      AtomicBoolean isLogCompaction = new AtomicBoolean(false);
```

Comment:
don't know why need use `AtomicBoolean`, I think there should be no concurrency here, reference old logic